layouts/default.vue

```diff
@@ -86,6 +86,7 @@ export default {
       { title: 'Logout', icon: 'mdi-logout', ref: null, logout: true, auth: true, menu: true },
       { title: 'Editar Regras', icon: 'assignment', ref: '/rules/edit', auth: true, menu: false },
       { title: 'Regras', icon: 'chrome_reader_mode', ref: '/rules', auth: null, menu: false },
+      { title: "Cadastrar Pela Tag", icon: 'chrome_reader_mode', ref: '/tag', auth: null, menu: false },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7513933)
```suggestion
      { title: "Cadastrar Pela Tag", icon: "chrome_reader_mode", ref: '/tag', auth: null, menu: false },
```